src/main/java/com/quadfour/dao/ITaskDAO.java

```diff
@@ -4,10 +4,10 @@
 import com.quadfour.dto.TaskDTO;
 
 public interface ITaskDAO {
-
-    boolean save(TaskDTO taskDTO) throws Exception;
+    // I know its nitpicky but this should probably be renamed to saveTask just to be clear what you are saving
+    boolean saveTask(TaskDTO taskDTO) throws Exception;
 
     Iterable<TaskDTO> fetchAll() throws Exception;
-
-    List<TaskDTO> fetchSpecimensByPlantId(int plantID);
+    // you must have copy pasted this from the plant application and forgot to change the name and return type
+    List<TaskDTO> fetchTaskByTaskId(int taskId);
```

Comment:
Yes, looks like copy paste, and this is an improvement.

Ditto comments above - don't put rationale in source code.  Save it for the pull request.